src/convertToObject.js

```diff
@@ -3,16 +3,38 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const transformedToSortedString = sourceString.split(';')
+    .map(str => str.trim())
+    .join('; ')
+    .split(': ')
+    .map(str => str.trim())
+    .join(': ')
+    .split(';')
+    .filter(element => element !== ' ' && element !== '')
+    .map(element => element.trim())
+    .sort((element1, element2) => {
```

Comment:
Too many methods, can it be simplified 🤔 

P.S. You can do it using only `split` and `reduce`